internal/serve/httphandler/health_handler.go

```diff
@@ -22,28 +25,60 @@ const (
 //
 // https://datatracker.ietf.org/doc/html/draft-inadarei-api-health-check-06#name-api-health-response
 type HealthResponse struct {
-	Status    Status `json:"status"`
-	Version   string `json:"version,omitempty"`
-	ServiceID string `json:"service_id,omitempty"`
-	ReleaseID string `json:"release_id,omitempty"`
+	Status    Status            `json:"status"`
+	Version   string            `json:"version,omitempty"`
+	ServiceID string            `json:"service_id,omitempty"`
+	ReleaseID string            `json:"release_id,omitempty"`
+	Services  map[string]Status `json:"services,omitempty"`
 }
 
 // HealthHandler implements a simple handler that returns the health response.
 type HealthHandler struct {
-	Version   string
-	ServiceID string
-	ReleaseID string
+	Version          string
+	ServiceID        string
+	ReleaseID        string
+	DBConnectionPool db.DBConnectionPool
+	Producer         events.Producer
+	EventBrokerType  events.EventBrokerType
```

Comment:
What do you think about adding a `Type()` or `BrokerType()` method to the producer? This way you ensure the producer has all the info needed for us to build its health status, rather than relying on an extra parameter.

Reply:
Addressed in d854b5f9e9ef197be9157b3b07c29e1a9f852fc7. 
Also added to Consumer